ETC.tsv

```diff
@@ -14724,7 +14724,7 @@ ETC_20151001_014725	Defeat the boss together with Hauberk!
 ETC_20151001_014726	Suspicious Device
 ETC_20151001_014727	Stop hiding and come out now!
 ETC_20151001_014728	Defeat Kartas's hunter!
-ETC_20151001_014729	Legywn family's vassal
+ETC_20151001_014729	Legwyn Family's Subordinate
```

Comment:
It can be a military position where a vassal usually isn't?
These subordinates aid Gailus Legwyn in a fight against a demon. It's in a quest at Mochia Forest :P